portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,4 +49,14 @@ private String convertToJsonUsingGson(ArrayList toConvert) {
     String json = gson.toJson(toConvert);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment-input");
+    messages.add(comment);
+
+    // Redirects to the current page to see new comment added.
+    response.sendRedirect("/index.html#connect-container");
```

Comment:
Can you create a private static final string for this as well and add a comment.

Reply:
I created a variable for the string. For adding a comment, I added a comment for the constants. Is that what you mean?

Reply:
Yes! Thank you